### ISSUE TITLE
--peer-port option seems to be ignored

### ISSUE THREAD

**Issue by ghaskins** [hyperledger-archives/fabric#1662]
It does not appear that the --peer-port option has the proper impact to configuration
## Description

Try to start the peer with a modified port via the command line and it seems to be ignored:

```
$ peer node start --peer-port=30302
12:27:55.413 [crypto] main -> INFO 001 Log level recognized 'info', set to INFO
12:27:55.436 [main] serve -> INFO 003 Security enabled status: false
12:27:55.437 [main] serve -> INFO 004 Privacy enabled status: false
12:27:55.435 [eventhub_producer] start -> INFO 002 event processor started
12:27:55.438 [chaincode] NewChaincodeSupport -> INFO 005 Chaincode support using peerAddress: 0.0.0.0:30303
12:27:55.519 [state] loadConfig -> INFO 006 Loading configurations...
12:27:55.521 [state] loadConfig -> INFO 007 Configurations loaded. stateImplName=[buckettree], stateImplConfigs=map[numBuckets:%!s(int=1000003) maxGroupingAtEachLevel:%!s(int=5) bucketCacheSize:%!s(int=100)], deltaHistorySize=[500]
12:27:55.521 [state] NewState -> INFO 008 Initializing state implementation [buckettree]
12:27:55.523 [buckettree] initConfig -> INFO 009 configs passed during initialization = map[string]interface {}{"bucketCacheSize":100, "numBuckets":1000003, "maxGroupingAtEachLevel":5}
12:27:55.539 [buckettree] initConfig -> INFO 00a Initializing bucket tree state implemetation with configurations &{maxGroupingAtEachLevel:5 lowestLevel:9 levelToNumBucketsMap:map[7:40001 2:13 6:8001 4:321 9:1000003 5:1601 3:65 1:3 0:1 8:200001] hashFunc:0xa79590}
12:27:55.542 [buckettree] newBucketCache -> INFO 00b Constructing bucket-cache with max bucket cache size = [100] MBs
12:27:55.559 [buckettree] loadAllBucketNodesFromDB -> INFO 00c Loaded buckets data in cache. Total buckets in DB = [23]. Total cache size:=3120
12:27:55.559 [genesis] loadConfigs -> INFO 00d Loading configurations...
12:27:55.560 [genesis] loadConfigs -> INFO 00e Configurations loaded: genesis=map[chaincodes:<nil>], mode=[], deploySystemChaincodeEnabled=[false]
12:27:55.561 [genesis] func1 -> INFO 00f No genesis block chaincodes defined.
12:27:55.566 [consensus/controller] NewConsenter -> INFO 010 Creating default consensus plugin (noops)
12:27:55.599 [consensus/noops] newNoops -> INFO 011 NOOPS consensus type = *noops.Noops
12:27:55.602 [consensus/noops] newNoops -> INFO 012 NOOPS block size = 500
12:27:55.602 [consensus/noops] newNoops -> INFO 013 NOOPS block timeout = 1s
12:27:55.606 [rest] StartOpenchainRESTServer -> INFO 014 Initializing the REST service on 0.0.0.0:5000, TLS is disabled.
12:27:55.607 [main] serve -> INFO 015 Starting peer with id=name:"jdoe" , network id=dev, address=0.0.0.0:30303, discovery.rootnode=, validator=true
```

Note that the peer still starts on port 30303.

**Comment by ghaskins** [comment]
Discovered while reviewing #1645

**Comment by binhn** [comment]
@jeffgarratt @lehors you re probably more familiar with these cmd flags;  please comment

**Comment by mastersingh24** [comment]
@ghaskins  - couldn't tell you if this was ever used, but since I started back in mid-December all "address" properties in the config are `host/ip:port` combos.

so my bet is that this can be safely removed since it does function as "designed" (does nothing) but not as "expected" (changes the port)

**Comment by ghaskins** [comment]
I am totally fine with this going either way (fixed or removed).  I mostly wanted to capture the issue while it was fresh on my mind.

**Comment by mastersingh24** [comment]
at this point, my vote would be to remove it

**Comment by kchristidis** [comment]
This can be closed, with #1645 in.
